getting-started/px-licensing.md

```diff
@@ -179,4 +179,32 @@ The "license file" will need to be uploaded to one of the PX-nodes, and activate
 Finally, please note that the license installation is a non-obtrusive process, which will not interfere with the data stored
 on the PX volumes, nor will it interrupt the active IO operations.
 
+**LICENSE TRANSFER**:
+
+From PX Enterprise 1.4.0 onwards, a valid enterprise license can be transferred between two Portworx clusters. Both Portworx clusters need to be operational at the time of license transer.
+The source cluster must have a valid PX-Enterprise license, while the destination cluster can have either a valid or expired Trial license, PX-Developer, or expired PX-Enterprise license.
```

Comment:
Please add 2 more details:
* the license transfer happens directly between the Portworx clusters, so at least one node from source cluster must have a network connectivity to a node in the target Portworx cluster.
* after the successful license transfer, the two Portworx clusters will have swapped identities, and licenses (for example, Portworx cluster A will have Trial license originally from Portworx cluster B, while the cluster B will have the PX Enterprise license originally from cluster A)